src/components/EmailAddresses.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+  export let emails;
+</script>
+
+{#each emails as email}
+  <span class="block">
+    <a href="mailto:{email}" target="_blank">{email}</a></span>
```

Comment:
Any reason this was set to target=blank_? I don't think it should be necessary (suspecting it got added by mistake see also discussion in #142)

```suggestion
    <a href="mailto:{email}">{email}</a></span>
```

Reply:
I thought it would be nice for the users to open their browser-based email app in a new tab without having to leave the current Glean page. However I read discussion #142, I understand the reasoning for keeping default behavior for links for now. Suggested change committed.